test(transformer): move named test to exports folder

Dunqing opened the pull request:
I made a mistake, I should have put it in the `exports`  folder and not `export`

Dunqing commented:
* **#7924** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7924?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#7923** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7923?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#7922** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7922?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7922?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`
<hr />

## How to use the Graphite Merge Queue

Add either label to this PR to merge it via the merge queue:
- <kbd>0-merge</kbd> - adds this PR to the back of the merge queue
- <kbd>hotfix</kbd> - for urgent hot fixes, skip the queue and merge this PR next

You must have a Graphite account in order to use the merge queue. Sign up using [this link](https://app.graphite.dev/invite/github/oxc-project?ref=merge-queue-instructions-in-stack-comment&prId=7922).

<h2></h2>
<sub>

An organization admin has enabled the [Graphite Merge Queue](https://graphite.dev/docs/graphite-merge-queue) in this repository.

</sub>

<sub>

Please do not merge from GitHub as this will restart CI on PRs being processed by the merge queue.

</sub>

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

overlookmotel reviewed (commented):
Diff also contains a change to `crates/oxc_semantic/src/builder.rs` which seems unrelated. I assume that's a mistake?

Dunqing commented:
> Diff also contains a change to `crates/oxc_semantic/src/builder.rs` which seems unrelated. I assume that's a mistake?

Yes, I've made another mistake 😢.

Dunqing committed:
test(transformer): move named test to exports folder (#7922)

I made a mistake, I should have put it in the `exports`  folder and not `export`

overlookmotel commented:
Ignore me. Answered my own question. Yes, it got changed in #7923 anyway.
